compose.env

```diff
@@ -5,9 +5,8 @@ OLLAMA_HOST=0.0.0.0
 
 
 ## INFINITY VARS
-INFINITY_EMBEDDING_MODEL=mixedbread-ai/mxbai-embed-large-v1
-INFINITY_RERANKING_MODEL=mixedbread-ai/mxbai-rerank-xsmall-v1
-INFINITY_BATCH_SIZE=4
+INFINITY_EMBEDDING_MODEL="mixedbread-ai/mxbai-embed-large-v1;mixedbread-ai/mxbai-rerank-xsmall-v1"
```

Comment:
We will keep them separate for the user to edit

Reply:
Okay!